osi_object.proto

```diff
@@ -264,6 +273,56 @@ message StationaryObject
             //
             COLOR_WHITE = 10;
         }
+
+        //
+        // \brief Attributes of type emitting structure
+        //
+        message EmittingStructureAttribute
+        {
+            // This message determines the range of the wavelength and its
+            // desired number of samples.
+            //
+            repeated WavelengthData wavelength_data = 1;
+
+            // Original intensities of emitted electromagnetic wave referred to a discrete
+            // wavelength defined in WavelengthData. This maximum value is weighted based on the IntensityDistribution.
+            //
+            // Unit: W/m^2
+            //
+            // \note max_intensity_per_wavelength.size() = WavelengthData.samples_number.size()
+            //
+            repeated double max_intensity_per_wavelength = 2;
+
+            // Distribution of the emitted electromagnetic wave's intensity based on the intensity_per_wavelength as
+            // maximum value.
+            //
+            repeated IntensityDistribution emitted_intensity_distribution = 3;
```

Comment:
**Output from CCB 09.06.2021**
Since this message defines the "FoV" of the emitting object and not the actual distribution of each "ray" a better name should be suggested (including the word Spatial either in the message name or in the definition).